src/index.html

```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <!-- Search bar -->
+    <div class="search-bar__content">
+      <div class="search-bar">
+        <input
+        class="search-bar__input
+        search-bar__input--size--large"
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles”">
```

Comment:
```suggestion
        <input
          class="search-bar__input
          search-bar__input--size--large"
          type="text"
          data-qa="big"
          placeholder="Try “Los Angeles”"
        >
```

Use this styling for all the cases.